src/main/java/com/ivoyant/DatabaseOperations.java

```diff
@@ -0,0 +1,195 @@
+                            if (ConnectionHandler.disconnect(connectionMap.get(key)))
+                                System.out.println("Disconnected\nThank You :)");
+                            else {
+                                connectionMap.get(key).close();
+                                System.out.println("Disconnected\nThank You :)");
+                            }
+                            return;
+                        default:
+                            System.out.println("Invalid Operation Chosen...");
+                            break;
+                    }
+                }
+            } else {
+                System.out.println("Connection Failed");
+            }
+        } catch (Exception e) {
+            System.out.println(e.getLocalizedMessage());
+        }
+    }
+
+    // Creating Table
+    public void createTable() {
+        System.out.print("Table Name : ");
+        String tableName = scanner.nextLine();
+        System.out.print("Column Names With Datatypes And Constraints : ");
+        String columnNames = scanner.nextLine();
+        if (TableOperations.createTable(connectionMap.get(key), tableName, columnNames) == -1) {
+            System.out.println("Error Occured While Creating Table Named " + tableName);
+        }
+    }
+
+    // Displaying Table
+    public void showTables() {
+        ResultSet resultSet = TableOperations.showTables(connectionMap.get(key));
+        if (resultSet != null) {
+            try {
+                while (resultSet.next()) {
+                    System.out.println("Table Name : " + resultSet.getString(1));
+                }
+            } catch (Exception e) {
+                System.out.println(e.getLocalizedMessage());
+            }
+        }
+    }
+
+    // Drop Table
+    public void dropTable() {
+        System.out.print("Enter Table Name To Drop : ");
+        String tableName = scanner.nextLine();
+        if (TableOperations.dropTable(connectionMap.get(key), tableName)) {
+            System.out.println(tableName + " Dropped Successfully");
+        } else {
+            System.out.println(tableName + " Drop Not Successful");
+        }
+    }
+
+    // Insert Data
+    public void insertData() {
+        System.out.print("Enter Table Name : ");
+        String tableName = scanner.nextLine();
+        System.out.print("Enter Column Names (Separated By ,): ");
+        String columns = scanner.nextLine();
+        System.out.print("Enter Values List (Separated By ,): ");
+        String values = scanner.nextLine();
+        int count = DataOperations.insertData(connectionMap.get(key), tableName, columns, values);
+        if (count == -1) {
+            System.out.println("Insertion UnSuccessful");
+        } else {
+            System.out.println("Inserted " + count + " Rows Successfully.");
+        }
+    }
+
+    // Display Data
+    public void displayData() {
+        System.out.print("Enter Column List : ");
```

Comment:
Does it work If I give column list as '*'?

Reply:
Yes it will work